packages/wix-ui-core/src/components/popover/Popover.tsx

```diff
@@ -230,7 +246,22 @@ export class Popover extends React.Component<PopoverProps, PopoverState> {
     }
   };
 
+  _onKeyDown = (e) => {
+    const { onEscPress } = this.props;
+
+    if (e.keyCode === 27) {
+      onEscPress();
```

Comment:
This prop might be undefined. 
```suggestion
      onEscPres && onEscPress();
```

Reply:
Lets pass the event too